crates/cdk-axum/Cargo.toml

```diff
@@ -16,3 +16,7 @@ cdk = { path = "../cdk", version = "0.4.0", default-features = false, features =
 tokio = { version = "1", default-features = false }
 tracing = { version = "0.1", default-features = false, features = ["attributes", "log"] }
 futures = { version = "0.3.28", default-features = false }
+moka = { version = "0.11.1", features = ["future"] }
+serde_json = "1.0.128"
+paste = "1.0.15"
+triomphe = "0.1.12"
```

Comment:
```suggestion
```

We don't need this here we can just pin it in the nix file. I think it will case the msrv build to fail since we pin to a version lower then the one here.